orchestra/bots/tests/test_assignment_policies.py

```diff
@@ -0,0 +1,43 @@
+from unittest.mock import patch
+
+from orchestra.models import Iteration
+from orchestra.models import Task
+from orchestra.tests.helpers import OrchestraTestCase
+from orchestra.tests.helpers.fixtures import setup_models
+from orchestra.utils.task_lifecycle import assign_task
+from orchestra.utils.task_lifecycle import create_subsequent_tasks
+from orchestra.utils.task_lifecycle import submit_task
+
+
+class StaffBotAutoAssignTestCase(OrchestraTestCase):
+
+    def setUp(self):
+        super().setUp()
+        setup_models(self)
+
+    @patch('orchestra.bots.staffbot.send_mail')
+    def test_preassign_workers(self, mock_mail):
+        project = self.projects['staffbot_assignment_policy']
+
+        # Create first task in test project
+        create_subsequent_tasks(project)
+        self.assertEqual(project.tasks.count(), 1)
```

Comment:
i assert that `mock_mail` is called, but we can check `StaffingRequests` as well